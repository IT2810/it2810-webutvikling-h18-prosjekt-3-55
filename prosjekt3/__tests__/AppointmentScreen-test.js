import 'react-native';
import React from 'react';
import AppointmentScreen from '../screens/AppointmentScreen';

import renderer from 'react-test-renderer';

test('snapshot settingsscreen', () => {
    const tree = renderer.create(<AppointmentScreen />).toJSON();
    expect(tree).toMatchSnapshot();
});