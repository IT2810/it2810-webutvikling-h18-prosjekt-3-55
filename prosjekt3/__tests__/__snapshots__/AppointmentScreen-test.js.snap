// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot settingsscreen 1`] = `null`;
